.github/workflows/test-indicators.yml

```diff
@@ -24,7 +24,7 @@ jobs:
     env:
 
       # identifying primary configuration so only one reports coverage
```

Comment:
This comment should probably read:

> identifying primary configuration so only one reports to console

Reply:
Thanks. applied ([b7a33dd](https://github.com/facioquo/stock-indicators-python/pull/386/commits/b7a33ddbcbb4481e07948d98a6376b1f5a7cde70))